pom.xml

```diff
@@ -201,11 +200,24 @@
           </execution>
         </executions>
         <groupId>org.springframework.boot</groupId>
+        <version>${spring-boot-maven-plugin.version}</version>
       </plugin>
       <plugin>
         <artifactId>jacoco-maven-plugin</artifactId>
-        <version>${jacoco-maven-plugin.version}</version>
         <groupId>org.jacoco</groupId>
+        <version>${jacoco-maven-plugin.version}</version>
+      </plugin>
+      <plugin>
```

Comment:
The Jib plugin is used for docker migration. Is this planned as well ?

Reply:
It was naively copied from the example I was using. I'll remove it. Development on this app is pretty much finished and we're moving onto the next project, so I don't think we intend to migrate it.